land_claims.civmap.json

```diff
@@ -89,9 +90,8 @@
+{"name":"Camiseo","shortname":"Camiseo","color":"#3d7de3","declutter":3,"x":-11170,"z":-3330,"id":"58c526f2-29dd-4fbb-8382-cf06aee77d05","polygon":[[[-10653,-3342],[-10593,-3356],[-10511,-3287],[-10472,-3326],[-10450,-3376],[-10368,-3430],[-10273,-3440],[-10033,-3440],[-9912,-3561],[-9826,-3594],[-9766,-3658],[-9813,-3758],[-9858,-3818],[-9965,-3771],[-10046,-3802],[-10216,-3816],[-10262,-3851],[-10384,-3906],[-10435,-3951],[-10499,-3971],[-10598,-3877],[-10671,-3788],[-10779,-3778],[-10880,-3802],[-10982,-3888],[-11022,-3916],[-11072,-3920],[-11186,-3857],[-11279,-3844],[-11355,-3745],[-11454,-3729],[-11564,-3740],[-11603,-3679],[-11708,-3608],[-11761,-3539],[-11820,-3561],[-11848,-3605],[-11989,-3651],[-12150,-3658],[-12120,-3456],[-12124,-3315],[-12156,-3207],[-12130,-2979],[-12188,-2943],[-12282,-2949],[-12321,-2963],[-12426,-3107],[-12478,-3162],[-12508,-3185],[-12539,-3206],[-12630,-3248],[-12677,-3303],[-12735,-3109],[-12768,-2903],[-12800,-2792],[-12700,-2748],[-12732,-2644],[-12636,-2524],[-12656,-2432],[-12588,-2391],[-12526,-2410],[-12462,-2396],[-12455,-2337],[-12436,-2285],[-12392,-2267],[-12360,-2221],[-12315,-2189],[-12273,-2185],[-12202,-2256],[-12178,-2300],[-12204,-2329],[-12167,-2384],[-12089,-2304],[-12041,-2322],[-12008,-2306],[-11889,-2283],[-11795,-2208],[-11699,-2238],[-11621,-2192],[-11312,-2486],[-10824,-2774],[-10683,-2960],[-10633,-3106],[-10645,-3233],[-10670,-3318]]]},
+{"name":"Cape Cocoa (Capeland)","shortname":"Capeland","color":"#7580ff","declutter":3,"x":3830,"z":650,"id":"04898976-6db9-41e9-834b-70d6c4578c3f","polygon":[[[3915,629],[3948,593],[3937,574],[3907,567],[3883,536],[3898,506],[3930,483],[3919,456],[3936,432],[3971,407],[3974,374],[3995,337],[3939,281],[3942,180],[3955,146],[3945,96],[3914,77],[3923,43],[3917,9],[3881,-14],[3807,-17],[3795,21],[3749,61],[3754,106],[3733,149],[3683,179],[3673,208],[3734,277],[3673,297],[3667,382],[3675,411],[3708,412],[3739,432],[3766,454],[3742,482],[3745,529],[3736,556],[3735,579],[3717,601],[3697,605],[3687,636],[3685,687],[3673,704],[3638,681],[3613,692],[3613,726],[3652,736],[3741,834],[3768,862],[3790,868],[3805,924],[3798,988],[3799,1010],[3832,1030],[3839,1045],[4002,1164],[4097,1202],[4185,1097],[4197,990],[4111,946],[4126,888],[4079,839],[4107,801],[4118,764],[4082,754],[4067,717],[4069,697],[4043,680],[4016,682],[3999,683],[3971,651],[3935,651]]]},
```

Comment:
Add n\ for Cape Cocoa (Capeland) 

Not sure if it should be done like this since () should be for exclaves, but I understand why you did.